libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -613,4 +631,42 @@ void AP_IOMCU::update_safety_options(void)
     }
 }
 
+/*
+  check ROMFS firmware against CRC on IOMCU, and if incorrect then upload new firmware
+ */
+bool AP_IOMCU::check_crc(void)
```

Comment:
Should be named `update_io_firmware` or similair. This does a lot more then just check the CRC.